etl/steps/data/garden/demography/2022-11-30/life_expectancy.py

```diff
@@ -395,3 +435,50 @@ def add_region_aggregates(frame: pd.DataFrame) -> pd.DataFrame:
     # concatenate
     df = pd.concat([frame, df]).sort_values(["country", "year"], ignore_index=True).drop(columns="population")
     return df
+
+
+def add_population_americas(df: pd.DataFrame):
+    pop = load_america_population_from_unwpp()
+    df = df.merge(pop, on=["country", "year"])
+    return df
+
+
+def load_america_population_from_unwpp():
+    """Load population data from UN WPP for Northern America and Latin America and the Caribbean.
+
+    We use this dataset instead of the long-run because we want the entities as defined by the UN.
+    """
+    # load population from WPP
+    locations = ["Latin America and the Caribbean (UN)", "Northern America (UN)"]
+    ds = Dataset(GARDEN_POPULATION_WPP)
+    df = ds["population"].reset_index()
+    df = df.loc[
+        (df["location"].isin(locations))
+        & (df["metric"] == "population")
+        & (df["sex"] == "all")
+        & (df["age"] == "all")
+        & (df["variant"].isin(["estimates", "medium"])),
+        ["location", "year", "value"],
+    ]
+    assert len(set(df["location"])) == 2, f"Check that all of {locations} are in df"
+    df["location"] = df["location"].replace(REGION_MAPPING)
+
+    # rename columns
+    df = df.rename(columns={"location": "country", "value": "population"})
+    return df
+
+
+def make_anomaly_table() -> Table:
```

Comment:
Oh, interesting way how to get arbitrary information across steps :). In `VariableMeta` there's a field `additional_info` that we use to carry around extra information (usually for internal purposes). You can put your dictionary there. I'm just not sure where since it lives under variable. Perhaps `life_expectancy_0`? We could also add `additional_info` to Table or Dataset if that would feel more natural.

If you feel that data annotations are close then don't bother rewriting this, we can refactor it once we have proper place for annotations.